### PR TITLE
Added installed apps support:

### PR DESCRIPTION
**Pull request by bijoshtj**
- Function to check a package is installed or not.
- Function to retrieve installation data of other apps (specified company app)

### CODE REVIEW

File: android/UtilsPlugin.java [hunk 1]
```diff
@@ -334,4 +358,51 @@ public void updateShortcutItems(String params) {
 			logger.log("{utils-native} Exception creating shortcut... " + ex);
 		}
 	}
+
+	public void getInstalledApps(String params) {
+                Settings settings = Settings.getInstance();
```
Comment (rampr) [inline android/UtilsPlugin.java]: indentation

File: android/UtilsPlugin.java [hunk 2]
```diff
@@ -334,4 +358,51 @@ public void updateShortcutItems(String params) {
...
+		JSONArray appsList = new JSONArray();
+
+                if (settings == null) {
+                        Settings.build(_context);
+                        settings = Settings.getInstance();
```
Comment (rampr) [inline android/UtilsPlugin.java]: indentation here too

Comment (rampr) [inline android/UtilsPlugin.java]: I think this file is using a mix of spaces and tabs

File: android/UtilsPlugin.java (hunk 1 above)
Comment (bijoshtj) [inline android/UtilsPlugin.java]: its correct. I followed intentation for this file. @rampr

**Pull request merged**